worlds/earthbound/__init__.py

```diff
@@ -0,0 +1,601 @@
+                                 f" Titanic Ant => {self.boss_list[2]}\n" +
+                                 f" Captain Strong => {self.boss_list[3]}\n" +
+                                 f" Everdred => {self.boss_list[4]}\n" +
+                                 f" Mr. Carpainter => {self.boss_list[5]}\n" +
+                                 f" Mondo Mole => {self.boss_list[6]}\n" +
+                                 f" Boogey Tent => {self.boss_list[7]}\n" +
+                                 f" Mini Barf => {self.boss_list[8]}\n" +
+                                 f" Master Belch => {self.boss_list[9]}\n" +
+                                 f" Trillionage Sprout => {self.boss_list[10]}\n" +
+                                 f" Guardian Digger => {self.boss_list[11]}\n" +
+                                 f" Dept. Store Spook => {self.boss_list[12]}\n" +
+                                 f" Evil Mani-Mani => {self.boss_list[13]}\n" +
+                                 f" Clumsy Robot => {self.boss_list[14]}\n" +
+                                 f" Shrooom! => {self.boss_list[15]}\n" +
+                                 f" Plague Rat of Doom => {self.boss_list[16]}\n" +
+                                 f" Thunder and Storm => {self.boss_list[17]}\n" +
+                                 f" Kraken => {self.boss_list[18]}\n" +
+                                 f" Guardian General => {self.boss_list[19]}\n" +
+                                 f" Master Barf => {self.boss_list[20]}\n" +
+                                 f" Starman Deluxe => {self.boss_list[21]}\n" +
+                                 f" Electro Specter => {self.boss_list[22]}\n" +
+                                 f" Carbon Dog => {self.boss_list[23]}\n" +
+                                 f" Ness's Nightmare => {self.boss_list[24]}\n" +
+                                 f" Heavily Armed Pokey => {self.boss_list[25]}\n" +
+                                 f" Starman Junior => {self.boss_list[26]}\n" +
+                                 f" Diamond Dog => {self.boss_list[27]}\n" +
+                                 f" Giygas (Phase 2) => {self.boss_list[28]}\n")
+
+        if self.options.dungeon_shuffle:
+            spoiler_handle.write("\nDungeon Entrances:\n")
+            for dungeon in self.dungeon_connections:
+                spoiler_handle.write(
+                    f" {dungeon} => {self.dungeon_connections[dungeon]}\n"
+                )
+        
+        if self.has_generated_output:
+            spoiler_handle.write("\nArea Levels:\n")
+            spoiler_excluded_areas = ["Ness's Mind", "Global ATM Access", "Common Condiment Shop"]
+            for area in self.area_levels:
+                if area not in spoiler_excluded_areas:
+                    spoiler_handle.write(f" {area}: Level {self.area_levels[area]}\n")
+
+    def create_item(self, name: str) -> Item:
+        data = item_table[name]
+        return Item(name, data.classification, data.code, self.player)
```

Comment:
World is missing an Item subclass overriding `game`. This should be "EBItem" or something along those lines.